client/src/components/Sidebar/SidebarPanel.scss

```diff
@@ -13,6 +13,7 @@
   z-index: 400;
   display: flex;
   flex-direction: column;
+  overflow: hidden;
```

Comment:
This was intentionally removed in 52739832bcfd0146c32de7ab9b38b5016feb29d6 (see https://github.com/wagtail/wagtail/pull/10330#discussion_r1171275271). Unfortunately, we have [a report on Slack](https://wagtailcms.slack.com/archives/C81FGJR2S/p1738676530813579) that reintroducing `overflow: hidden` here broke nested menu groups.

Now I guess we know why the `overflow: hidden` was necessary. I think ideally we should fix the issue mentioned in this PR in a way that doesn't break multi-level viewset groups #10786. I'll play around and report back.

Reply:
Thanks @laymonage